apps/etl/etl_tasks/emdat.py

```diff
@@ -0,0 +1,13 @@
+from celery import shared_task
+
+from apps.etl.extraction.sources.emdat.extract import import_hazard_data
+from apps.etl.transform.sources.emdat import transform_emdat_data
+
+
+@shared_task
+def extract_and_transform_emdat_data(**kwargs):
+    # Extract the data from emdat
+    extraction_id = import_hazard_data()
+
+    # Transform the data from emdat
+    transform_emdat_data(extraction_id)
```

Comment:
Any reason we aren't running this in separate celery tasks?

Reply:
@thenav56 I did this so that transformation starts only after the extraction ends. The whole extraction object is saved in a row. Thus pass extraction_id into transformation_emdat() and do the required transformation.